### PR TITLE
Removed constants module mess.

### PR DESCRIPTION
**Pull request by nekokatt**
### Checklist

- [ ] I have run `nox` and all the pipelines have passed.
- [ ] I have made unittests according to the code I have added/modified/deleted.

### CODE REVIEW

File: hikari/impl/shard.py [hunk 1]
```diff
@@ -87,6 +88,20 @@
 # Rate-limit for chunking requests (used to prevent saturating the entire
 # ratelimit window).
 _CHUNKING_RATELIMIT: typing.Final[typing.Tuple[float, int]] = (60.0, 60)
+# Supported gateway version
+_VERSION: int = 6
```
Comment (davfsa) [inline hikari/impl/shard.py:92]: This gets defined here but never used. Is that intentional?

**Review by davfsa — DISMISSED** [review]
Looks good to me, just had a little question

File: hikari/impl/shard.py (hunk 1 above)
Comment (nekokatt) [reply hikari/impl/shard.py:99]: nope, fixed.

**Review by davfsa — APPROVED** [review]

File: hikari/events/channel_events.py [hunk 2]
```diff
@@ -585,6 +586,7 @@ class PrivatePinsUpdateEvent(PinsUpdateEvent, PrivateChannelEvent):
     # <<inherited docstring from ChannelEvent>>.
 
     last_pin_timestamp: typing.Optional[datetime.datetime] = attr.ib(repr=True)
+
```
Comment (FasterSpeeding) [inline hikari/events/channel_events.py:589]: Are these gaps intentional?

Comment (nekokatt) [reply hikari/events/channel_events.py:589]: Apparently so.

**Pull request merged**